src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -71,15 +71,9 @@ def setup(self, builder: Builder):
         self.population_view = builder.population.get_view(self.columns_used)
         self.businesses = None
 
-        job_change_rate_data = builder.lookup.build_table(data_values.YEARLY_JOB_CHANGE_RATE)
-        self.job_change_rate = builder.value.register_rate_producer(
-            f"{self.name}.job_change_rate", source=job_change_rate_data
-        )
+        self.job_change_rate = builder.lookup.build_table(data_values.YEARLY_JOB_CHANGE_RATE)
+        self.businesses_move_rate = builder.lookup.build_table(data_values.BUSINESS_MOVE_RATE_YEARLY)
 
-        move_rate_data = builder.lookup.build_table(data_values.BUSINESS_MOVE_RATE_YEARLY)
-        self.businesses_move_rate = builder.value.register_rate_producer(
```

Comment:
When we talked about this last week we discussed how `register_rate_producer` is actually doing something important (namely scaling the rate for a time-step). We only want to change this in the locations where we are only using `register_value_producer`. 

Both the job change rate and business move rates should remain as they were.